### PR TITLE
[feature] Connects register form with backend - OT102-65

### PR DESCRIPTION
**Pull request by JonatanGarbuyo**
[OT102-65](https://alkemy-labs.atlassian.net/browse/OT102-65)

## Description
AS: User
I WANT: enter the site already logged in, automatically
TO: navigate the site quickly

## Acceptance criteria:
When the user registers on the site, he should automatically log in and
redirect to base path (/). To do this you need to modify the server
to return the token once the user is created and store the same
on the client side

#### validation error
validations

### CODE REVIEW

File: src/pages/auth/Register.js [hunk 1]
```diff
@@ -17,24 +19,84 @@ import {
...
+
+const passwordChars = /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[^a-zA-Z0-9]).{8,}$/
+
+const registerSchema = Yup.object().shape({
+  firstName: Yup.string().required('Nombre es Obligatorio'),
+  lastName: Yup.string().required('Apellido es Obligatorio'),
+  email: Yup.string().email('Formato del email inválido').required('Required'),
+  password: Yup.string()
+    .required('Required')
+    .min(6, 'Contraseña debe tener al menos 6 caracteres')
```
Comment (gabodmc) [inline src/pages/auth/Register.js]: Fijate que el pass desde el back viene con 8 caracteres minimos. Y tmb habria que agregarle que necesita una mayuscula, minuscula, numero y caracter especial.

**Review by gabodmc — CHANGES_REQUESTED** [review]
Yo lo veo re bien... solo un comentario

File: src/pages/auth/Register.js (hunk 1 above)
Comment (JonatanGarbuyo) [reply src/pages/auth/Register.js]: si. yo le agregue el de mayuscula, minuscula, numero y caracter especial  que no lo tenia.
ahi le cambio el 6

**Review by gabodmc — APPROVED** [review]

**Review by dvelezroman — APPROVED** [review]
excelente

**Review by gabyMangoni — APPROVED** [review]

**Pull request merged**